js/script.js

```diff
@@ -138,17 +138,26 @@ async function getLastCommitDate(repo, timeBlock) {
 }
 
 repos.addEventListener('click', function(event) {
+    let timeOfClick = new Date().getTime();
+    let timeDiff = (timeOfClick - event.target.nextElementSibling.dataset.LastRequestTime)>60000;
+
     if (event.target.tagName === 'P') {
```

Comment:
this is a little bit strange logic: if (event.target.tagName === 'P') {

better to use some class or id, but not tag, because html tag will not give you understanding what the logic is, but some class name like 'repository-item' could be more clear and useful